cloudsdk/src/main/java/io/particle/android/sdk/cloud/DeviceState.java

```diff
@@ -33,6 +33,10 @@
     final String status;
     final String name;
     final boolean isConnected;
+    final boolean cellular;
```

Comment:
if these are nullable (and I bet they are, e.g.: IMEI on a Photon), the fields should be marked as such